### DISCUSSION TITLE
Community Meeting - Saturday, November 9, 2024

### DISCUSSION THREAD

**Discussion by wrashi** [GothenburgBitFactory/taskwarrior#3675] [Community Meetings]
Our bi-weekly community meeting happens TOMORROW! Saturday, November 9.

Starting time is 1600 UTC. 
Here is a link where you can check out what that means in your timezone: [World Clock Meeting Times](https://www.timeanddate.com/worldclock/meetingtime.html?month=11&day=9&year=2024&p1=5&p2=3904&p3=3918&p4=3921&iv=0).
The meeting details are below.

Google Meet joining info
Video call link: https://meet.google.com/tns-suqq-iaj
Or dial: ‪(CA) +1 289-323-1423‬ PIN: ‪830 535 232‬#
More phone numbers: https://tel.meet/tns-suqq-iaj?pin=8726596308824

To get a sense of what the meeting is like, you can check out some of the latest meetings here:
[TaskWarrior Community Meetings YouTube Playlist](https://www.youtube.com/playlist?list=PLU-esvACzq9XBiGjDCQnMQigKrjlNBq2W).

**Comment by linuxcaffe** [comment]
Is this thing still on?

**Comment by linuxcaffe** [comment]
I'm hoping to join that meeting, and if I understand correctly, it was to
start half an hour ago, but I'm all alone in that room.. ?

**Comment by djmitche** [comment]
It's starting now!

**Comment by djmitche** [comment]
And yes, we did get the time wrong. Apologies!

**Comment by wrashi** [comment]
The seasonal time change caught Dustin and me out this morning. The community meeting got going at the wrong time without realizing it until much too late! Sorry everyone!

The meeting was small, but took a terrific turn when a surprise TaskWarrior OG dropped in and a wide-ranging discussion ensued.
Check it out!
https://youtu.be/twJQ8Uw-bms
